examples/direct_fidelity_estimation.py

```diff
@@ -379,21 +381,25 @@ def direct_fidelity_estimation(
     n_qubits = len(qubits)
 
     clifford_circuit = True
-    clifford_state: Optional[cirq.CliffordState] = None
+    clifford_tableau: Optional[cirq.CliffordTableau] = None
```

Comment:
The tableau can be instantiated here directly, removing the need to wrap it in Optional. Any reason to instantiate with `None` first?

Reply:
Done.